src/Demo.tsx

```diff
@@ -0,0 +1,173 @@
+  }
+
+  makeColorList(colors: {}): JSX.Element {
+    return (
+      <ul className="clearfix" style={{margin: "0", backgroundColor: `${this.state.lights ? "#fff" : "#000"}`}}>
+        { Object.entries(colors).map(color => this.makeColorSwatch(color)) }
+      </ul>
+    );
+  }
+
+  makeColorSwatch(info: string[]): JSX.Element {
+    const liStyle = {
+      display: "flex",
+      float: "left",
+      flexDirection: "column",
+      alignItems: "center",
+      margin: "20px",
+      color: `${this.state.lights ? "#000" : "#fff"}`
+    };
+    const swatch = {
+      height: "140px",
+      width: "140px",
+      clipPath: "polygon(20% 0%, 80% 0%, 100% 20%, 100% 80%, 80% 100%, 20% 100%, 0% 80%, 0% 20%)",
+      display: "flex",
+      alignItems: "center"
+    };
+    const hexCodeStyle = {
+      display: "block",
+      width: "100%",
+      padding: "30px 0",
+      textAlign: "center",
+      color: "#000",
+      background: "rgba(255, 255, 255, 0.7)",
+      clipPath: "polygon(100% 50%, 0% 0%, 0% 50%, 100% 100%)"
+    };
+    let style = {...swatch, ...{background: info[0]}};
+    return (<li className="clearfix" style={liStyle}>
+              <label>{info[1]}</label>
+              <div style={style}><span style={hexCodeStyle}>{info[0]}</span></div>
+            </li>
+          );
+  }
+
+  setError() {
+    this.setState({lights: this.state.lights, errorText: "ERROR", successText: null, infoText: null});
+  }
+  setSuccess() {
+    this.setState({lights: this.state.lights, errorText: null, successText: "SUCCESS", infoText: null});
+  }
+  setInfo() {
+    this.setState({lights: this.state.lights, errorText: null, successText: null, infoText: "INFO"});
+  }
+
+  render(): JSX.Element {
+
+
+    return(
+      <div id="demo">
+        <Header text="Welcome to the demo page!" imgSrc="./logo.png" logOut="#" />
+        <div className="list">
+          <Panel headerText="The default panel" body={<span>Your panel content goes here!</span>} />
+          <Panel headerText="Success panel" style="success" body={<span>Your panel content goes here!</span>} />
+          <Panel headerText="Warning panel" style="warning" body={<span>Your panel content goes here!</span>} />
+          <Panel headerText="Danger panel" style="danger" body={<span>Your panel content goes here!</span>} />
+          <hr></hr>
+          <section style={{display: "flex", justifyContent: "space-between", width: "42%", margin: "20px"}}>
+            <SubmitButton callback={this.setError} content="Error message" />
+            <SubmitButton callback={this.setSuccess} content="Success message" />
+            <SubmitButton callback={this.setInfo} content="Info message" />
+          </section>
+          <Form
+            title="This is a form"
+            onSubmit={() => alert("You have submitted the form!")}
+            content={[<Fieldset key="fieldset" legend="This is a fieldset" elements={[<Input key="input" label="An input field" name="name"/>]}/>]}
```

Comment:
Long lines like these are hard to read. In general lines should be shorter and something like this would benefit from being a variable to be easier to read.